www/sirm02/sp2/js/main.js

```diff
@@ -0,0 +1,160 @@
+$(document).ready(() => {
+
+    //whet reset, set url to default
+    history.pushState(null, 'Flicker recent', '../sp2/');
+
+
+    const appContainer = $(`#app`);
+    const tagsContainer = $('<nav>').addClass('tags');
+
+    const tags = ['airport', 'apple', 'bridge', 'car', 'cat', 'czechia', 'dog', 'landscape', 'smartphone', 'winter'];
+
+
+    const tagsForm = $('#tagsForm');
+    const tagsInput = $('input[name="tagsInput"]');
+
+    tagsForm.submit((event) => {
+        event.preventDefault();
+        const inputValue = tagsInput.val();
+        const tagsNames = inputValue.replace(' ', ',');
+
+
+        if (!tags.includes(tagsNames)) {
+            tags.push(tagsNames);
+            createButton(tagsNames);
+
+            $('.tagButton').unbind();
+
+            $('.tagButton').click(function () {

```

Comment:
tagButton vybrat predem

Reply:
na toto koukám, ale nejde mi najít způsob, jak to zařídit...

Reply:
```js
const tagButton = $('.tag-button');
...
tagButton....
tagButton....
```
?

Reply:
přesně tohle nefunguje..

Reply:
a je vubec potreba nabindovat tlacitka opetovne? nestaci to udelat jen pro nove pridane tagy, vraci funkce createButtons neco?